litellm/integrations/s3.py

```diff
@@ -122,8 +124,9 @@ def log_event(self, kwargs, response_obj, start_time, end_time, print_verbose):
                     pass
 
             s3_object_key = (
+                (self.s3_path.rstrip('/') + '/' if self.s3_path else '') + 
```

Comment:
any reson why s3 path should be part of the key ?

Reply:
I believe that is the way you point to a folder in boto3. 

Could not find better docs then here.. 

https://stackoverflow.com/questions/39272397/uploading-file-to-specific-folder-in-s3-using-boto3#comment65879979_39272397